doc/ref_model/chapters/chapter03.rst

```diff
@@ -0,0 +1,1052 @@
+================ =============================================================================
+
+**Table 3-2:** Attributes of compute resources
+
+Virtual Storage
+^^^^^^^^^^^^^^^
+
+A virtual machine and container can consume storage through a number of means. These include storage that is:
+
+-  managed via the hypervisor and container runtime (Hypervisor Attached for virtual machine and Container Persistent for containers) and is connected via cloud infrastructure underlay network and
+-  Shared File Storage and the Object storage which is connected via the tenant / user overlay network.
+   The details of the tenant storage consumption model are covered in section `"3.6.3 Storage for Tenant Consumption" <#3.6.3>`__.
+
+In managing the provision of virtual storage the tenant should be able to request alternate performance levels, capacity and behaviours. The set of selectable attributes includes:
+
+-  Storage class: Block, File, Object.
+-  Retention Policy - persistent (storage volume / data) is persistent across stop/start of workload; ephemeral storage - there is no data retention across stop/start events for the workload.
+-  Underlying physical device type (HDD, SSD, etc.).
+-  Performance characteristic - defined as: Latency, IOPS (Input/Output Operations per second), and throughput.
+-  Enhanced features - set of selectable features such as: auto-replicate, encryption, snapshot support.
+
+Note that approximate numeric ranges for the qualitative values used above are given in the
+`Storage Extensions <./chapter04.md#4.2.6>`__ section.
+
+Storage resources have the following attributes, with metric definitions that support verification through passive measurements (telemetry) where appropriate:
+
+========================= ==============================================================================================
+Attribute                 Description
+========================= ==============================================================================================
+``name``                  name of storage resources
+``data retention policy`` persistent or ephemeral
+``performance``           Read and Write Latency, The average amount of time to perform a R/W operation, in milliseconds
+\                         Read and Write IOPS, The average rate of performing R/W in IO operations per second
+\                         Read and Write Throughput, The average rate of performing R/W operations in Bytes per second
+``enhanced features``     replication, encryption
+``type``                  block, object or file
+``size``                  size in GB, telemetry includes the amount of free, used, and reserved disk space, in bytes
+========================= ==============================================================================================
+
+**Table 3-3:** Attributes of storage resources
+
+Virtual Network
+^^^^^^^^^^^^^^^
+
+This topic is covered in `Network <#3.5>`__ section.
```

Comment:
This won't work. I think we need a fix for internal and external links.

Reply:
Fixed in b6a5d55